man/pdfpcrc.in

```diff
@@ -123,6 +123,11 @@ shown. This option enables a workaround (bool, Default is false).
 .B next-height
 Percentage of the height of the presenter screen to be used for the next slide (int, Default is 70).
 .TP
+.B next-slide-first-overlay
+If the next slide is an overlay group, show the first slide of that group in the
+next slide view instead of the last slide of that group (bool, Default is false).
+This option is ignored if "final-slide" is enabled.
```

Comment:
I don't think it should be ignored if `final-slide` is set. In that case, the next slide is still shown when the last overlay slide is reached, and so again it might be desired to show not the last but the first slide of the next-page overlay.

Reply:
That is a good suggestion and actually not that hard to implement.
I removed that line from the documentation and changed the implementation accordingly. Can you please review the changes? Thanks!